sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3318,6 +3318,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val BASE_64_CHUNKING = buildConf("spark.sql.base64.chunking")
```

Comment:
We are very careful about the configuration namespace. In this case, it's a little too much to add a new namespace, `spark.sql.base64.*` for this single configuration. Maybe, (1) `spark.sql.chunkBase64String` or (2) `spark.sql.chunkBase64String.enabled`? I prefer (1).

Reply:
This part, `val BASE_64_CHUNKING`, also should be revised accordingly.